ingesters/SimpleRelay/config.go

```diff
@@ -147,6 +150,33 @@ func verifyConfig(c *cfgType) error {
 			return fmt.Errorf("Listener %s preprocessor invalid: %v", k, err)
 		}
 	}
+	for k, v := range c.RegexListener {
```

Comment:
should this also attempt to compile the regex?  seems like we should catch a bad regex here.

Reply:
looks like it just needs to call Validate on v?